sensing/src/main/java/com/google/android/sensing/impl/SensorManagerImpl.kt

```diff
@@ -46,7 +50,11 @@ import kotlinx.coroutines.sync.withLock
 import kotlinx.coroutines.withContext
 import timber.log.Timber
 
-internal class SensorManagerImpl(context: Context, private val database: Database) : SensorManager {
+internal class SensorManagerImpl(
+  context: Context,
```

Comment:
do you need this at all if this is not used anywhere else in this class?

Reply:
I am thinking of merging SensingEngine APIs (CRUD APIs) into SensorManager to make it a unified interface. So 3 APIs will be moved:-
1. getCaptureInfo (captureId)
2. getCaptureInfo (externalIdentifier)
3. deleteDataInCapture(captureId)

Will need context in implementations of these APIs.